test/testcafe/spec/EnrollAuthenticatorGoogleAuthenticator_spec.js

```diff
@@ -74,8 +77,12 @@ test
     await enrollGoogleAuthenticatorPageObject.goTomanualSetup();
     await t.expect(enrollGoogleAuthenticatorPageObject.form.getTitle()).eql('Set up Google Authenticator');
     await t.expect(enrollGoogleAuthenticatorPageObject.isEnterCodeSubtitleVisible()).notOk();
-    await t.expect(enrollGoogleAuthenticatorPageObject.getmanualSetupSubtitle()).eql('Can\'t scan barcode?');
-    await t.expect(enrollGoogleAuthenticatorPageObject.getSharedSecret()).eql('ZR74DHZTG43NBULV');
+    await t.expect(enrollGoogleAuthenticatorPageObject.getmanualSetupSubtitle()).contains('Can\'t scan');
```

Comment:
Should the subtitle have changed? Or was this a mistake?

Reply:
So, in g2, it's "Can't scan barcode?"
In g3, we changed it to "Can't scan", which is why I changed the subtitle.
Don't want to have separate checks for g2 vs g3.

Reply:
Ohh, I thought we made the titles match completely between the two. No worries this makes sense. 🙏